meta/style-guide/README.md

```diff
@@ -0,0 +1,171 @@
+# Evergreen Style Guide
+
+This document serves to provide a set of guidelines for how to write good
+documentation for Evergreen. This guide is less focused on the content and
+more focused on the style and structure of the documentation.
+
+## People
+
+Whenever referring to someone in Evergreen, if they are a member of our
+team and have an entry in [purduehackers/dark-forest](https://github.com/purduehackers/dark-forest)
+then you should ALWAYS use the following format to refer to them:
+
+✅ Correct
+
+```markdown
+[@rayhanadev](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/rayhanadev.md)
+```
+
+You should never not link to someone in dark-forest if they have an entry there
+and you should never refer to someone by a handle besides their dark-forest
+handle.
+
+❌ Incorrect
+
+```markdown
+@rayhanadev
+@someotherhandleforray
+[Ray](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/rayhanadev.md)
+```
+
+## Linking
+
+When linking to other pages in the Evergreen documentation, you should always
+use absolute links from the root of the repository. This is to ensure that
+links do not break _toooooo_ hard when the documentation is moved around.
+
+✅ Correct
+
+```markdown
+[Hack Bishops](/meta/structure/bishops.md)
+```
+
+❌ Incorrect
+
+```markdown
+[Hack Bishops](../structure/bishops.md)
+[Hack Bishops](structure/bishops.md)
+```
+
+You should also always use the title of the page as the text for the link
+if it is a page in the Evergreen documentation.
+
+✅ Correct
+
+```markdown
+[Hack Bishops](/meta/structure/bishops.md)
+```
+
+❌ Incorrect
+
+```markdown
+[The Bishops Page](/meta/structure/bishops.md)
+```
+
+## Files and Folders
+
+As long as it generally makes sense, you should always feel free to create
+new files and folders in the Evergreen repository. Some general tips for when
+to create new files and folders:
+
+- If a page is getting too long, consider splitting it into multiple pages
+- If a page is getting too complex, consider splitting it into multiple pages
+- If aspects of a page can stand alone, consider splitting them into multiple pages
+- If you are working on a project, consider creating a folder for that project
+
+✅ Correct
+
+```plaintext
+/meta/structure/bishops.md
+/meta/structure/divisons.md
+/meta/structure/hierarchy.md
+/meta/structure/roles.md
+```
+
+❌ Incorrect
+
+```plaintext
+/meta/structure.md
+/meta/roles-and-divisons.md
+```
```

Comment:
This example is not communicated well in the above text. The above text says "feel free to create files, but split them out if they're too complex." The "feel free to create files" part confuses the rule. I think this rule should be simplified to something like: "Keep each page focused to one topic; do not make a file too broad or cover more than one specific topic"